Lib/fontTools/misc/xmlWriter.py

```diff
@@ -153,12 +153,12 @@ def stringifyattrs(self, *args, **kwargs):
 		for attr, value in attributes:
 			if not isinstance(value, (bytes, unicode)):
 				value = str(value)
-			data = data + ' %s="%s"' % (attr, escapeattr(value))
+			data = data + ' %s="%s"' % (str(attr), escapeattr(value))
 		return data
 
 
 def escape(data):
-	data = tostr(data, 'utf_8')
+	data = tostr(data, 'utf_8', errors='xmlcharrefreplace')
```

Comment:
`xmlcharrefreplace` seems reasonable, but why that `str(attr)`? The `attr` variable is for dict keys which cannot be anything but strings already -- or am I missing something else?